packages/java/parser-jvm-plugin-nonnull/src/main/java/dev/hilla/parser/plugins/nonnull/NonnullPlugin.java

```diff
@@ -51,29 +52,35 @@ public void enter(NodePath<?> nodePath) {
 
     @Override
     public void exit(NodePath<?> nodePath) {
-        if (!(nodePath.getNode().getTarget() instanceof Schema)) {
-            return;
-        }
-
-        var schema = (Schema<?>) nodePath.getNode().getTarget();
-
-        // Apply annotations from package (NonNullApi)
-        var annotations = getPackageAnnotationsStream(nodePath);
-
-        // Apply from current node, if source is annotated
-        if (nodePath.getNode().getSource() instanceof AnnotatedModel) {
-            annotations = Stream.concat(annotations,
-                    ((AnnotatedModel) nodePath.getNode().getSource())
-                            .getAnnotationsStream());
+        var node = nodePath.getNode();
+        if (node.getTarget()instanceof Schema schema) {
```

Comment:
Hmm, as far as I know, we cannot use that syntax in the production code yet since we need to support Java 11.
```suggestion
        if (node.getTarget() instanceof Schema) {
          var schema = node.getTarget();
```

Reply:
Are we going to cherry-pick to old releases?

Reply:
Hmm, I'm confused 😅 @platosha, what would you say?